src/main/java/com/github/ylegat/uncheck/Uncheck.java

```diff
@@ -68,4 +77,32 @@ public static void uncheck(CheckedOperation operation) {
             throw new RuntimeException(e);
         }
     }
+
+    /**
+     * <p>Wrap a transformation function.</p>
+     * <p>Exceptions are handled this way:</p>
+     * <ul>
+     * <li>RuntimeException are just propagated</li>
+     * <li>IOException are wrapped inside UncheckedIOException</li>
+     * <li>Other exceptions are wrapped inside RuntimeException</li>
+     * </ul>
+     *
+     * @param transformer the transformation function to apply
+     * @param <T>      the parameter type of the transformation function
+     * @param <R>      the result type of the transformation function
+     * @return a wrapper around the transformation function that handles exceptions in a lambda-friendly way.
+     */
+    public static <T, R> Function<T, R> uncheck(CheckedFunction<T, R> transformer) {
+        return value -> {
+            try {
+                return transformer.apply(value);
```

Comment:
I think we can reuse the logic of `uncheck(CheckedSupplier<T> supplier)` by just doing this:
```java
    public static <T, R> Function<T, R> uncheck(CheckedFunction<T, R> transformer) {
        return value -> uncheck(() -> transformer.apply(value));
    }
```